### DISCUSSION TITLE
Documentation Template

### DISCUSSION THREAD

**Discussion by DahyeonWoo** [euphony-io/euphony#240] [Ideas]
Each folder in euphony/euphony/src/main/core will need a description file (README.md).
I want to make and share a documentation template, what should I do?
Can I fill out the yaml file and apply it?

**Comment by designe** [comment]
@DahyeonWoo Great :) 
If you want to share documentation template, please leave it here :)
Markdown is only possible to make a documentation in Github.
Github is not supported to documentation template maker by yaml.

If you want to propose a documentation creation, please make an issue for documentation.
https://github.com/euphony-io/euphony/issues/new?assignees=&labels=documentation&template=documentation.yaml&title=%5BDocumentation%5D+%3Ctitle%3E

**Comment by DahyeonWoo** [comment]
I checked. Thank you.
